Makefile

```diff
@@ -107,6 +107,12 @@ build: # Build REANA client and cluster components.
 	pip uninstall -y reana-commons reana-client reana-cluster reana-db pytest-reana && \
 	reana-dev install-client && \
 	reana-dev install-cluster && \
+	if [ "${DEBUG}" -gt 0 ]; then \
```

Comment:
If DEBUG is guaranteed to be numerical, we could leave out the double quotes, here and in the other statement, I guess... Although this works well too.